chain/network/src/peer_manager/peer_manager_actor.rs

```diff
@@ -335,8 +298,20 @@ impl Actor for PeerManagerActor {
             (MONITOR_PEERS_INITIAL_DURATION, max_interval),
         );
 
-        // Periodically starts connected peer stats querying.
-        self.monitor_peer_stats_trigger(ctx, self.config.peer_stats_period);
+        // Periodically recompute PEER_CONNECTIONS metric.
+        let network_state = self.state.clone();
+        let period = self.config.peer_stats_period.try_into().unwrap();
+        actix::spawn(async move {
+            loop {
+                let mut m = HashMap::new();
+                let connected_peers = network_state.connected_peers.read();
+                for p in connected_peers.values() {
+                    *m.entry((p.peer_type, p.stats.load().encoding)).or_insert(0) += 1;
+                }
+                metrics::set_peer_connections(m);
+                tokio::time::sleep(period).await;
```

Comment:
`tokio::interval` would perhaps be a more pedantically correct option here, as it avoids clock drift. Doesn't really matter though.

Reply:
done